### PR TITLE
Update following changes in the oM for PlanarSurface

### PR DESCRIPTION
**Pull request by kThorsager**
### NOTE: Depends on 

https://github.com/BHoM/BHoM/pull/861
https://github.com/BHoM/BHoM_Engine/pull/1761
   
### Issues addressed by this PR
Required as part of other PRs

Updates to the new constructor for PlanarSurface

### Test files

Not needed as no functional changes

### CODE REVIEW

**Comment by kThorsager** [comment]
Noticed that there were no automatic reviewers here.

But hopefully this one will be less controversial

File: Rhinoceros_Engine/Convert/FromRhino.cs [hunk 1]
```diff
@@ -591,15 +591,17 @@ private static BHG.PlanarSurface ToBHoMPlanarSurface(this RHG.BrepFace face)
                 return null;
 
             BHG.PlanarSurface bhs = rhSurf.FromRhino() as BHG.PlanarSurface;
+            List<BHG.ICurve> internalBoundaries = bhs.InternalBoundaries.ToList();
```
[the remark this answers is not in the document]
Comment (kThorsager) [reply Rhinoceros_Engine/Convert/FromRhino.cs:594]: Doesn't sound unreasonable, but don't know how to test, so will trust you here

[the remark this answers is not in the document]
Comment (pawelbaran) [reply Rhinoceros_Engine/Convert/FromRhino.cs]: May be worth re-running tests from https://github.com/BHoM/Rhinoceros_Toolkit/pull/122 and https://github.com/BHoM/Rhinoceros_Toolkit/pull/126

[the remark this answers is not in the document]
Comment (kThorsager) [reply Rhinoceros_Engine/Convert/FromRhino.cs]: Gave it a shot and seems to be fine

**Pull request merged**